app/uk/gov/hmrc/cgtpropertydisposalsfrontend/testonly/JourneyStatusController.scala

```diff
@@ -119,7 +119,7 @@ class JourneyStatusController @Inject() (
   ): IncompleteSingleDisposalTriageAnswers =
     IncompleteSingleDisposalTriageAnswers(
       answers.individualUserType,
-      answers.numberOfProperties,
+      answers.numberOfProperties.isDefined,
```

Comment:
it more accurate to have `answers.numberOfProperties.contains(NumberOfProperties.One)` here